kolibri/plugins/coach/assets/src/views/plan/LessonResourceSelectionPage/ContentCardList.vue

```diff
@@ -16,14 +16,23 @@
         :aria-selected="contentIsChecked(content)"
       >
         <KCheckbox
-          v-if="contentHasCheckbox(content)"
+          v-if="contentHasCheckbox(content) && !showRadioButtons"
           class="content-checkbox"
           :label="content.title"
           :showLabel="false"
           :checked="contentIsChecked(content)"
           :indeterminate="contentIsIndeterminate(content)"
           @change="handleCheckboxChange(content, $event)"
         />
+        <KRadioButton
+          v-else-if="contentHasCheckbox(content) && showRadioButtons"
```

Comment:
Non-blocking, minor bug:

When I select the quiz and hit the confirm button, the KRadioButton disappears briefly before redirecting. It is _very_ pronounced on "Fast 3G" throttling.

Reply:
I think maybe "just use this on Fast 3G for a bit and fix what you see" could be a solid theme for a cohack, some things I noticed before I couldn't handle it anymore:

_Note, I have my cache disabled as well, so all typically cached API calls & resources would likely be much faster, but this is worth polishing_

- Click "Add section" -- the section tab is added... then nothing for a few seconds... then the slideout appears w/ the section settings
- Click "Add questions" -- basically the same, nothing happens - the page doesn't start spinning... it just sits there until the new content is displayed
- Basically a lot of transitions are met with a blank stare and sudden change rather than a spinner or some other indication of "something is happening just wait a sec"

Again -- not blocking, just noticed during testing here and can put into follow-up today